recipe/meta.yaml

```
@@ -37,11 +37,17 @@ requirements:
     - autograd-gamma >=0.3
 
 test:
+  requires:
+    - pytest >=3.3.0
   imports:
     - lifelines
     - lifelines.datasets
     - lifelines.fitters
     - lifelines.utils
+  commands:
+    - ls ${SRC_DIR}
+    - pytest ${SRC_DIR}/lifelines/tests/ --verbose --pyargs lifelines -k test_requirements
```

You probably only need this?

```suggestion
    - pytest --verbose --pyargs lifelines -k test_requirements
```

---

ah I've been trial & erroring to make this work. With what you suggested, it doesn't run _any_ tests.

---

let me try again

---

https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=90502&view=logs&jobId=1bf226d3-0e2f-52d8-fa93-7d9e633347b3&taskId=ec8d466d-e3b4-5115-4f06-222398f91e6c&lineStart=808&lineEnd=809&colStart=1&colEnd=1

---

okay, I believe I need to move my tests into my `lifelines` folder. I'll keep working on this 👍